src/util/Timer.cpp

```diff
@@ -376,7 +381,8 @@ VirtualClock::crank(bool block)
         size_t const WORK_BATCH_SIZE = 100;
         std::chrono::milliseconds const MAX_CRANK_WORK_DURATION{200};
 
-        YieldTimer ioYt(*this, MAX_CRANK_WORK_DURATION, WORK_BATCH_SIZE);
+        YieldTimer ioYt(*this, *mIODebtTracker, MAX_CRANK_WORK_DURATION,
```

Comment:
in addition to tracking time spent in `run_one` (as it's IO work), there might be edge cases when we should not call it at all (because we're processing the execution queue/timers).

Also, I don't know if there are situations where we don't want to call it because we maxed out both IO and execution queue but want to still trigger timers (some form of busy wait).